### PR TITLE
Color: Gray Value Updates

### PR DESCRIPTION
**Pull request by somethiiing**
Hex values for gray700/800/870/900 have been updated:

gray700 → #5E5952

gray800 → #363432

gray870 → rgba(94, 89, 82, 0.7)

gray900 → #262625

 

Add 

gray270 → rgba(228, 219, 211, 0.7)

white70 → rgba(255, 255, 255, 0.7)

white40 → rgba(255, 255, 255, 0.4)

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -5,6 +5,9 @@ on:
     branches:
       - main
 
+permissions:
```
Comment (zprong) [inline .github/workflows/release.yml:8]: What does this line do?

File: .changeset/slow-carrots-shop.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@cambly/syntax-icons": major
```
Comment (zprong) [inline .changeset/slow-carrots-shop.md:2]: Why does this need to be major?

**Review by btorndorff — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (somethiiing) [reply .github/workflows/release.yml:8]: was part of an old commit

File: .changeset/slow-carrots-shop.md (hunk 2 above)
Comment (somethiiing) [reply .changeset/slow-carrots-shop.md:2]: was part of an old commit

**Pull request merged**